src/plugins/mpris/item.h

```diff
@@ -0,0 +1,48 @@
+// albert - a simple application launcher for linux
+// Copyright (C) 2014-2015 Manuel Schneider
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+#include <QVariant>
+#include <vector>
+using std::vector;
+#include "abstractobjects.hpp"
+#include "player.h"
+using MPRIS::Player;
+#include <QDBusMessage>
+
+namespace Template {
```

Comment:
Oops, yea I deleted this file and thought later that it was not so bad after all to have an own class for this, so I copied the template and missed the namespace. 
The weird namespace has surprised me though at some point. :D
Going to fix this right away